src/test/php/org/bovigo/vfs/vfsStreamWrapperTestCase.php

```diff
@@ -211,12 +211,22 @@ public function is_executable()
      *
      * @test
      */
-    public function directoriesAndNonExistingFilesAreNeverExecutable()
-    {
-        $this->assertFalse(is_executable($this->fooURL));
-        $this->assertFalse(is_executable($this->fooURL . '/.'));
-        $this->assertFalse(is_executable($this->barURL));
-        $this->assertFalse(is_executable($this->barURL . '/.'));
+    public function directoriesAndNonExistingFilesAreSometimesExecutable()
+    {
+        // Inconsistent behavior has been fixed in 7.3
+        // see https://github.com/php/php-src/commit/94b4abdbc4d
+        if (PHP_VERSION_ID >= 70300) {
```

Comment:
Should we add a note about this in our readme?

Reply:
Yea I think this is good to know. Would the wiki be better since that's where the rest of the documentation is?

Reply:
Sure wiki is fine, it is a known issue that people should be aware of. We are just following the php behavior here.

Reply:
There was already a note in the ["Known Issues" page](https://github.com/bovigo/vfsStream/wiki/Known-Issues) about this so I've [updated its description](https://github.com/bovigo/vfsStream/wiki/Known-Issues/_compare/b07deb47a7a0f448d707430a8e684c58ab7c5da2...34f12a1cb5d3f5029e0dc80d4789f6f298d8a50d).